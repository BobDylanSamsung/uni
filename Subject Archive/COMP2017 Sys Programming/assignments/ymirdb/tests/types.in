set a 1 2 3
type a
set b 1 2 3
type b
set b a
type b
pop b
type b
set c a b
type c
pop c
pop c
type c
append c a b
type c
pop c
pop c
push c 1 2 3
type c
push c a
type c
bye
