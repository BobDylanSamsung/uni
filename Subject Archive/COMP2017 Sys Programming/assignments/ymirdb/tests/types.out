ok

simple

ok

simple

ok

general

a

simple

ok

general

a

b

simple

ok

general

a

b

ok

simple

ok

general

bye
